orchestrator/src/indexer/rooch.ts

```diff
@@ -287,27 +287,44 @@ export default class RoochIndexer extends Indexer {
       signer: this.keyPair,
     });
 
+    const notify_module = data.notify?.split("::") ?? [];
+
     log.info({
       address: Secp256k1Keypair.fromSecretKey(keeper_key.privateKey).getRoochAddress().toHexAddress(),
       target: data.notify ?? "",
       oracleAddress: this.oracleAddress,
-      receipt,
+      receipt: receipt.execution_info,
+      notify_module: data.notify?.split("::"),
     });
     try {
       if ((data.notify?.length ?? 0) > 66) {
-        const tx = new Transaction();
-        tx.callFunction({
-          target: data.notify ?? "",
-          args: [],
-        });
-        const notification_receipt = await client.signAndExecuteTransaction({
-          transaction: tx,
-          signer: Secp256k1Keypair.fromSecretKey(keeper_key.privateKey),
+        const module_abi = await client.getModuleAbi({
+          moduleAddr: notify_module[0] ?? "",
+          moduleName: notify_module[1] ?? "",
         });
-        log.info({ notification_receipt });
+        const function_abi = module_abi.functions.find(
+          (func) => func.name === notify_module[2] && !func.params.includes("&signer"),
+        );
+        if (function_abi) {
+          const tx = new Transaction();
+          tx.callFunction({
+            target: data.notify ?? "",
+            args: function_abi?.params?.length === 0 ? [] : [Args.objectId(data.request_id)],
```

Comment:
I see - https://linear.app/usherlabs/issue/LABS-1083/move-oracle-rooch-wants-requestid-argument-to-the-callback-notify